acquire/acquire.py

```diff
@@ -632,19 +632,40 @@ class Recents(Module):
     ]
 
 
+def recyclebin_filter(path: fsutil.TargetPath) -> bool:
+    return bool(path.stat().st_size >= (10 * 1024 * 1024))  # 10MB
+
+
 @register_module("--recyclebin")
+@module_arg("--large-files", action="store_true", help="Add files larger than 10MB in the Recycle Bin", default=False)
```

Comment:
```suggestion
@module_arg("--large-files", action="store_true", help="Collect files larger than 10MB in the Recycle Bin", default=False)
```



Maybe it's good to also add an inverse to _not_ collect any data files. E.g. `--no-data-files`.

There may be scenario's where for privacy reasons you will not want to collect actual data files.

Reply:
@Zawadidone can you still address this?

Reply:
Yes

Reply:
Can the logs message be adjusted to `log.info("Acquiring recycle bin from %s (%s)", fs, mountpoints)`?

Reply:
Yeah that seems fine.